Business/DataTransferObjects/CommentCreationDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Business.DataTransferObjects;
+
+public class CommentCreationDto
+{
+    public string Name { get; set; } = default!;
```

Comment:
If client don't include some field, it won't pass the default validation. Otherwise, if we allow user to pass null value (or omit those fields), it will be mapped to the entity with nulls and exception will be thrown during isertion into the db

Reply:
You can use validators such as FluentValidation for DTOs